packages/fluentui/accessibility/src/behaviors/Tree/treeTitleAsOptionBehavior.ts

```diff
@@ -0,0 +1,33 @@
+import * as _ from 'lodash';
+
+import { Accessibility } from '../../types';
+import { treeTitleBehavior, TreeTitleBehaviorProps } from './treeTitleBehavior';
+
+/**
+ * @description
+ * Adds role 'option' if the title is a leaf node inside the tree
+ * Adds 'aria-selected' to a leaf treeTitle if it is selectable, based on the 'selected' property
+ */
+export const treeTitleAsOptionBehavior: Accessibility<TreeTitleBehaviorProps> = props => {
+  const behavior = treeTitleBehavior(props);
+
+  const definition = _.merge(behavior, {
```

Comment:
I think we need be careful here which aria attribute we will apply on the option role.
https://www.w3.org/TR/wai-aria-1.1/#option

Because looks like, "aria-expanded", "aria-level" is not applicable on role=option.

Reply:
I just pushed a new change. Instead of `_.merge`, using below logic:
```js
  const definition = {
    ...behavior,
    attributes: {
```
And removed "aria-expanded", "aria-level" from this behaviour